File: beginner/pagina01/1008/1008.rs
use std::io;


fn main(){

    let mut a = String::new();
    let mut b = String::new();
    let mut c = String::new();

    io::stdin().read_line(&mut a).unwrap();
    io::stdin().read_line(&mut b).unwrap();
    io::stdin().read_line(&mut c).unwrap();

    let numero_funcionario: u32 = a.trim().parse::<u32>().unwrap_or_default();
    let horas_trabalhadas: u32 = b.trim().parse::<u32>().unwrap_or_default();
    let salario_por_hora: f32 = c.trim().parse::<f32>().unwrap_or_default();

    let salario: f32 = ( horas_trabalhadas as f32 ) * salario_por_hora;

    println!("NUMBER = {}", numero_funcionario);
    println!("SALARY = U$ {:.2}", salario);
}

File: beginner/pagina01/1008/1008.cs
using System;
using System.Globalization;

public class URI
{
    public static void Main(string[] args)
    {
        Console.WriteLine("NUMBER = {0}", Console.ReadLine());

        int worked_hours;
        double receive_per_hour, salary;

        worked_hours = int.Parse(Console.ReadLine());
        receive_per_hour = double.Parse(Console.ReadLine());

        salary = receive_per_hour * worked_hours;

        Console.WriteLine("SALARY = U$ {0}", 
            salary.ToString("F2", CultureInfo.InvariantCulture));
    }
}